src/index.test.ts

```diff
@@ -1,6 +1,7 @@
 import dotenv from "dotenv";
 import { ethers } from "ethers";
 import { AutomateSDK } from "./lib";
+import { TriggerType } from "./types/Trigger.interface";
```

Comment:
could import from `./types` directly

Reply:
fixed in d99d312655f980f580bb981a4c607d35f34eec09